python/lsst/ts/ATWhiteLightSource/chillerModel.py

```diff
@@ -347,10 +510,10 @@ def tempParser(self, msg):
         """
         temp = 0
         temp += int(msg[1:4])
-        temp += int(msg[4])/10
+        temp += int(msg[4]) / 10
         if msg[0] == "-":
             temp = 0 - temp
-        return(temp)
+        return temp
```

Comment:
I strongly suggest replacing all this with:
```
return int(msg) / 10
```
(Note that in Python 3 `/` reliably returns a float).